docs/syntax.md

```diff
@@ -134,7 +134,36 @@ export let name;
 | Special Characters           | `&nbsp;`                                 | `{'\xa0'}` or `{String.fromCharCode(160)}`         |
 | Attributes                   | `dash-case`                              | `camelCase`                                        |
 
-### TODO: Styling
+### URL resolution
+
+It’s important to note that Astro **won’t** transform your URLs for you. For example, say you have the following structure:
+
+```
```

Comment:
(I have my "concise docs" / "readers aren't reading all that" hat on from the other PRs) Can we make these docs more concise, and communicate the point faster? Here would be an example attempt:

---

### URL resolution

It’s important to note that Astro **won’t** transform your URL references inside of HTML for you. For example:

```html
<!-- src/components/button.astro -->
<!-- INCORRECT: This will render this exact HTML to every page its used on, 
        no matter what the URL is. Relative URLs will almost always break on some pages.  -->
<a href="./foo.css">
```

Instead, you should choose one of two options:

#### Option 1: Absolute URL References

Use absolute URLs to files in the `public/` directory whenever possible to get around this limitation. Absolute URLs will always point to the same file, regardless of what page they are rendered on.

```html
<!-- src/components/button.astro -->
<a href="/css/foo.css">
```

#### Option 2: Asset Import References

Most assets should live in the `public/` directory so that they are available via absolute URL. If you absolutely need to reference an asset inside of the `src/` directory, you can import a reference via JavaScript ESM Import:


```svelte
---
// src/components/button.astro 
// NOTE: This will only work for images, videos, and non-code assets.
import imageUrl from './cat.png';
---
<!-- src/components/button.astro -->
<img src={imageUrl}>
```

Reply:
I’m confused when you ask to “make these docs more concise” but you increase the word count from what I’ve written by 10% (with the same number of code blocks). Did you mean to say “add more explanation and add more headings to make it more referenceable?”

Reply:
I guess I got a bit carried away there in my example :) To be fair I did add some new docs as well (Option 2)

Maybe there's a better word/phrase for what i'm trying to communicate with these docs changes. "to the point" or maybe even "easy to jump into"?

If it helps, the way that I think about it is "It should be easy to do a control-F find on the page, and then understand the example in front of them" Fun-to-read prose, diff snippets that build on previous examples, setting up a file system at the top of the page and then assuming the reader has seen it, etc. are getting the scalpel.

Reply:
Feel free to make edits on top of mine, especially if you think it could be improved! The structure of the headings, code snippets, and overall layout of information is really the suggestion I was trying to make.

Reply:
OK that makes sense—thanks for clarifying!

Reply:
As an extra note, it is hard to receive feedback where things are completely rewritten, including bullet points, tone, language, organization, etc to the point the original writing is completely gone. For me, at least, it’s easier to understand docs feedback where you only make specific edits to the text while keeping the same structure and tone.

Or if you have more sweeping structural feedback, a general “break this up into sections” comment is more helpful than receiving a full rewrite in the form of several paragraphs.